hasher-matcher-actioner/src/OpenMediaMatch/blueprints/development.py

```diff
@@ -14,17 +14,14 @@
     lookup_signal,
     lookup_signal_with_distance,
 )
-from OpenMediaMatch.utils.flask_utils import api_error_handler
+from OpenMediaMatch.utils.flask_utils import api_error_handler, str_to_bool
 
 from OpenMediaMatch.utils import dev_utils
 from OpenMediaMatch import persistence
 from OpenMediaMatch.storage.postgres.flask_utils import reset_tables
 
 
-from threatexchange.exchanges.fetch_state import CollaborationConfigBase
```

Comment:
unused

Reply:
We don't use the whole `query_media` function I don't see it being used anywhere.